Source/Core/Core/HW/AudioInterface.cpp

```diff
@@ -122,8 +122,8 @@ void DoState(PointerWrap& p)
   p.Do(s_sample_counter);
   p.Do(s_interrupt_timing);
   p.Do(s_last_cpu_time);
-  p.Do(s_ais_sample_rate);
-  p.Do(s_aid_sample_rate);
+  p.Do(s_ais_sample_rate_divisor);
+  p.Do(s_aid_sample_rate_divisor);
```

Comment:
Please update the savestate version in State.cpp, otherwise loading an old savestate will completely mess up the sample rates.